src/server/config/schema.js

```diff
@@ -30,9 +30,9 @@ module.exports = () => Joi.object({
     exclusive: Joi.boolean().default(false)
   }).default(),
 
-  uuid: Joi.string().guid().default(),
 
   server: Joi.object({
+    uuid: Joi.string().guid().default(),
```

Comment:
Shouldn't I just add a note about it in `docs/releasenotes.asciidoc`?

@epixa does that suffice?